### ISSUE TITLE
Lack of implementation of proposed solutions

### ISSUE THREAD

**Issue by reshad-equinix** [boucadair/rfc3535-20years-later#18]
New solutions proposed by WGs such as NETMOD and NETCONF very often lack an implementation or only have a partial implementation. The situation has improved with the last hackathons (e.g. for YANG-Push) but these solutions became RFCs without a known implementation:
- YANG-Push
- Schema-mount
- NMDA

Schema-mount allegedly has only 1 known implementation because of the complexity of the solution. That means the IETF most likely spent lots of cycles for something which won't be deployed ever.

While hackathons have improved the situation, I wonder if we should mandate at least 1 implementation. For open-source, `sysrepo`/`libyang` are decent choices.

**Comment by boucadair** [comment]
Added  point in #21 as this is a valid point to discuss. 

Note however that I added the following as this is a recurrent story in the IETF.

NEW-OPS-REQ-READILTY-IMPLEM:
: It is tempting to consider mandating at least one implementation. However, there were areas which imposed in the past rules for implementations/management for I-Ds to be published as PS (e.g., {{?RFC1264}}), but these rules were relaxed for reasons described, e.g., {{?RFC4794}} and left it to the WGs to decide about the actual measures to put in place. To date, only IDR WG has clear guidance on two implementations.
